### PR TITLE
CSS/HTML Changes and Added Extras Page and Link

### PR DESCRIPTION
**Pull request by EDrumheller**
**##Changed:**
- Changed the Login HTML and CSS to make it match our other pages!
- Added some < title > tags to the pages.
- Updated the < nav >
- Added Colin's HTML pages, though removed "Facts" will use for "Extras" page instead. (Easter Egg)
- Cleaned up the CSS file (removed or merged redundant CSS)
- Added some World-Wide Wastage facts to the extras page.
- Edited header to make the Logo clickable to take user to extras page. (Easter Egg)

**##To-Do:**
- Working on adding facts to extras page.
- Working on adding sessions!

### CODE REVIEW

File: javascript/StoreItem.js [hunk 1]
```diff
@@ -3,6 +3,22 @@ var fruit_apple = new Image();
 fruit_apple.src = '../images/apple.png'; 
 
 
+<<<<<<< HEAD
```
Comment (dwloiter) [inline javascript/StoreItem.js]: you need to resolve conflict correctly.

File: javascript/StoreItem.js [hunk 2]
```diff
@@ -54,14 +70,23 @@ function StoreItem(posX, posY, width, height) {
 //The function below returns a Boolean value representing whether the point with the coordinates supplied "hits" the particle.
 StoreItem.prototype.hitTest = function (hitX, hitY) {
     return (hitX > this.x && hitX < this.x + this.width && hitY > this.y && hitY < this.y + this.height);
+>>>>>>> master
 }
 
 //A function for drawing the particle.
 StoreItem.prototype.drawToContext = function (theContext) {
+<<<<<<< HEAD
```
Comment (dwloiter) [inline javascript/StoreItem.js]: you need to resolve conflict correctly.

File: javascript/draggable.js [hunk 3]
```diff
@@ -36,6 +36,74 @@ function canvasApp() {
 	}
 	
 	function makeShapes() {
+<<<<<<< HEAD
```
Comment (dwloiter) [inline javascript/draggable.js]: you need to resolve conflict correctly.

File: javascript/script.js [hunk 4]
```diff
@@ -1,73 +1,157 @@
...
-        success: function (data)          //on recieve of reply
-        {
-            foodDatas = data;
-        }
-    });
-});
-
-/* Run main menu screen */
-main_menu();
+<<<<<<< HEAD
```
Comment (dwloiter) [inline javascript/script.js]: you need to resolve conflict correctly.

**Comment by EDrumheller** [comment]
Ah I thought I got rid of all of those. I saw them on some pages after pulling from master. Thx for finding them!

**Pull request closed without merging**